src/Namotion.Reflection/Infrastructure/DynamicApis.cs

```diff
@@ -42,16 +48,33 @@ static DynamicApis()
 
         /// <summary>Gets a value indicating whether XPath APIs are available.</summary>
         public static bool SupportsXPathApis => XPathExtensionsType != null;
+#else
+        /// <summary>Gets a value indicating whether file APIs are available.</summary>
+        public static bool SupportsFileApis => true;
```

Comment:
Keep the properties and always return true to have a common public api and avoid breaking changes?

Reply:
Yes, that was the goal, not breaking anything. I don't know the project, and I don't know the implication of a change of the API here

Reply:
Ah sorry it is already like that - long day